src/target/lpc55xx.c

```diff
@@ -420,14 +420,12 @@ static bool lpc55xx_enter_flash_mode(target_s *target)
 	// put it back into a known state. Unfortunately target_reset hangs for this
 	// target and I'm not sure why, so the below is a viable workaround for now.
 
-	const uint32_t reg_pc_value = LPC55xx_CODE_PATCH_ADDRESS | 1;
+	const uint32_t reg_pc_value = LPC55xx_CODE_PATCH_ADDRESS | 1U;
 
 	// Execute a small binary patch which just disables interrupts and then hits
 	// a breakpoint, to allow the flash IAP calls to run undisturbed. This patch
 	// consists of the instructions CPSID I; BKPT; in ARM Thumb encoding.
-	const uint32_t CODE_PATCH = 0xbe00b672U;
-
-	target_mem32_write32(target, LPC55xx_CODE_PATCH_ADDRESS, CODE_PATCH);
+	target_mem32_write32(target, LPC55xx_CODE_PATCH_ADDRESS, 0xbe00b672U);
```

Comment:
Are you sure that having a magic number instead of named constant better here for readability?

Reply:
ok, so what are you suggesting. Should we leave this as is and you will polish it out later?

Reply:
Was suggesting we'd push a fix with the other stuff we'd already done to the file - as have just done 🙂

Reply:
Fantastic. I will approve and merge it after the GHA finish. :)